### ISSUE TITLE
Input multiple sub.BAM files for a single sample

### ISSUE THREAD

**Issue by phillip-richmond-alamya** [epi2me-labs/wf-human-variation#121]
### Is your feature related to a problem?

The latest dorado basecaller is producing a few thousand sub.bam files into a folder, which I then need to merge into a single BAM file before starting EPI2ME. It would be lovely if I can skip that step external to the wf-human-variation pipeline. Is that possible?

### Describe the solution you'd like

Allow input_bam_folder as a parameter as opposed to a single .bam file. 

### Describe alternatives you've considered

I can merge the bam files myself prior to running wf-human-variation

### Additional context

Would be very helpful and save time!

Thanks!

**Comment by RenzoTale88** [comment]
@phillip-richmond-alamya thanks for the feedback, we will have a look into this.

**Comment by RenzoTale88** [comment]
@phillip-richmond-alamya we added support to multiple BAM for a single sample as of v2.1.0.
